requirements.txt

```diff
@@ -31,3 +31,4 @@ termcolor==1.1.0
 urllib3==1.25.9
 Werkzeug==1.0.1
 wrapt==1.12.1
+MIDIUtil
```

Comment:
If possible, it would be nice to have the exact version specified here. Use `pip show midiutil` to find it! If you're using a virtual environment, you can also simply freeze your venv by using `pip freeze > requirements.txt`